### DISCUSSION TITLE
Carousel component request

### DISCUSSION THREAD

**Discussion by nisalk96** [heroui-inc/heroui#679] [Feature requests]
NextUI is beautiful 😍 modern UI library
It would be better if it's have a Carousel/Slide show component (feature request)
with horizontal and vertical nav 😎
Eg:

https://ant.design/components/carousel/#header
https://getuikit.com/docs/slideshow

**Comment by aKus-dev** [comment]
Yes, please, that would be great

**Comment by LeonGeldsch** [comment]
Commenting since it's been some time since the original request and I would still like to see this!

**Comment by steeeved** [comment]
Yes please. Give us Carousel/Slide

**Comment by Pedro-Laurenti** [comment]
yes i agree
